### PR TITLE
fix #69810 milcow.com

### LINKED ISSUE #69810: milcow.com
### Issue URL (Incorrect Blocking)
 [http://milcow.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fmilcow.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 10
AdGuard product: | AdGuard for Android v4.0 nightly 9 (48)
Browser: | Chrome (data compression)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#69810

### CODE REVIEW

File: JapaneseFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/66610
 @@||identity.mparticle.com/*/login$xmlhttprequest,domain=dplay.jp
+! https://github.com/AdguardTeam/AdguardFilters/issues/69810
+@@||*.jp/banner$domain=milcow.com
```
Comment (zloyden) [inline JapaneseFilter/sections/whitelist.txt:7]: `@@||myfigure.jp/banner/$image,domain=milcow.com` or change it to more specifc.
@Yuki2718 should we unblock it on the website?

Comment (Yuki2718) [reply JapaneseFilter/sections/whitelist.txt:7]: Probably yes, kinda promoted items but not obtrusive ads. But this site is a portal to show and compare various shopping sites so `@@||*.jp/banner$domain=milcow.com` makes sense, the contents will switch as a matter of time. OTOH you can block `milcow.com##.adaida`.

Comment (stanislav-atr) [reply JapaneseFilter/sections/whitelist.txt:7]: Yeah, .adaida is obv ad leftover, removed.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@ cinematoday.jp##.col-sidebar > div[style="min-height:100px"]
 dailyanime.news##.advTable
 dailyanime.news##.side-layoutLMovable
 dailyanime.news##.side-layoutRMovable
+milcow.com##td > div.middle_rank > div.adaida
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:21]: Why not just `milcow.com##.adaida`?

File: JapaneseFilter/sections/whitelist.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/whitelist.txt:7]: Ok, but I don't see any other requests, except the `myfigure.jp`.
@stanislav-atr please also change the exception.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**